presto-docs/src/main/sphinx/release/release-0.228.rst

```diff
@@ -0,0 +1,18 @@
+=============
+Release 0.228
+=============
+
+General Changes
+_______________
+* Reduce excessive memory usage by ExchangeClient
+* Improve coordinator stability on parsing functions that may create large constant arrays/maps/structs (e.g., `SEQUENCE`, `REPEAT`, `ARRAY[...]`, etc) by delaying the evaluation on these functions on workers.
+* Optimization provided for LIMIT 0 queries
+
+Hive Changes
+____________
+* Streamline ORC read path for varints and fixed width numbers.
```

Comment:
Is this a user facing change? It sounds like an internal refactoring.

Reply:
Should we just mention performance improvements?

Reply:
oh yeah definitely. didn't even realize this was a performance improvement.